hybridse/include/vm/physical_op.h

```diff
@@ -759,22 +759,22 @@ class PhysicalSimpleProjectNode : public PhysicalUnaryNode {
     ColumnProjects project_;
 };
 
-class PhysicalAggrerationNode : public PhysicalProjectNode {
+class PhysicalAggregationNode : public PhysicalProjectNode {
```

Comment:
@SohamRatnaparkhi you also need to change `PhysicalAggrerationNode` in all other files.

Reply:
Yes, sure! I will make the changes. So should I create a new PR or just commit changes to the existing PR.


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
@SohamRatnaparkhi you can continue on this pr.